code/game/turfs/open/floor/misc_floor.dm

```diff
@@ -204,8 +204,13 @@
 	desc = "Corrupted steel."
 	icon_state = "plating_rust"
 
-/turf/open/floor/plating/rust/plasma
-	initial_gas_mix = "plasma=104;TEMP=293.15"
+/turf/open/floor/plating/plasma/initial_gas_mix = ATMOS_TANK_PLASMA
+
```

Comment:
```suggestion
/turf/open/floor/plating/plasma
	initial_gas_mix = ATMOS_TANK_PLASMA
```
I think the original syntax used was a lot clearer than this

Reply:
absolute-pathed vars are not a good idea.

Reply:
do this